Pod/Classes/URBNAlertView.m

```diff
@@ -151,18 +161,43 @@ - (instancetype)initWithAlertConfig:(URBNAlertConfig *)config alertStyler:(URBNA
         
         // Button Constraints
         self.buttons = [btns copy];
+        self.sectionCount++;
         if (self.buttons.count == 1) {
-            self.sectionCount++;
             [buttonContainer addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"H:|-btnLeftMargin-[btnOne]-btnRightMargin-|" options:0 metrics:metrics views:@{@"btnOne" : self.buttons.firstObject}]];
             [buttonContainer addConstraints:[NSLayoutConstraint constraintsWithVisualFormat:@"V:|-btnTopMargin-[btnOne(btnH)]-btnBottomMargin-|" options:0 metrics:metrics views:@{@"btnOne" : self.buttons.firstObject}]];
         }
-        else if (self.buttons.count == 2) {
-            self.sectionCount++;
+        else if (self.buttons.count == 2 && !self.alertStyler.useVerticalLayoutForTwoButtons.boolValue) {
```

Comment:
after looking at this it does-- if there are 2 buttons and `self.alertStyler.useVerticalLayoutForTwoButtons` is true, it goes to the next block which adds the separators